### DISCUSSION TITLE
Deforum errors

### DISCUSSION THREAD

**Discussion by yarabsahel** [TheLastBen/fast-stable-diffusion#737] [Q&A]
Hi Ben,

I never been able to get the script working with Deforum, but after reading your comment yesterday that it's ready, I get this error when using the v2-786 model. It only starts working for 1% and then stops with this error. Please advise; thank you.
* Settings 3D/2D, soundtrack added ON/Off, resolution 768x768:

 File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions/deforum-for-automatic1111-webui/scripts/deforum_helpers/depth.py", line 135, in predict
    depth_map = midas_depth*anim_args.midas_weight + adabins_depth*(1.0-anim_args.midas_weight)
TypeError: Concatenation operation is not implemented for NumPy arrays, use np.concatenate() instead. Please do not rely on this error; it may not be given on all Python implementations.

**Comment by TheLastBen** [comment]
You should check the Deforum repo if this issue is present everywhere or it's just a colab issue
